docs/dev_plan.md

```diff
@@ -0,0 +1,307 @@
+(dev-plan)=
+
+# OpenTurbine Development Plan
+
+:::{warning}
+This page is under construction. Check back here throughout FY23
+for updates, and see activity at https://github.com/exawind/openturbine.
+:::
+
+## Background and overview
+
+OpenTurbine development started in early 2023 with primary funding from the
+U.S. Department of Energy (DOE) Wind Energy Technologies Office and with
+additional funding from the DOE Exascale Computing Project (ECP). It is being
+developed by researchers at the National Rewable Energy Laboratory and Sandia
+National Laboratories.
+
+OpenTurbine is meant to be an open-source wind turbine structural dynamics
+simulation code designed to meet the research needs of WETO and the broader
+wind energy community for land-based and offshore wind turbines. OpenTurbine
+will provide high-fidelity, highly performant structural dynamics models that
+can couple with low-fidelity aerodynamic/hydrodynamic models like those in
+OpenFAST, and high-fidelity computational fluid dynamics (CFD) models like
+those in the WETO and Office of Science supported ExaWind code suite.
+OpenTurbine will be designed to address shortcomings of wind turbine structural
+models and codes that are so important to the success of WETO modeling efforts.
+
+## Development priorities and use cases
```

Comment:
I covered most of this and expanded on it below. @michaelasprague I think we should either remove this section in favor of "Design Drivers and Considerations" or at least incorporate it there.